spec/docx/smoke/api_stroke_spec.rb

```diff
@@ -6,4 +6,13 @@
     docx = builder.build_and_parse('js/docx/smoke/api_stroke/get_class_type.js')
     expect(docx.elements[1].nonempty_runs.first.text).to eq('Class Type = stroke')
   end
+
+  it 'ApiRGBColor | ToJSON method' do
+    docx = builder.build_and_parse('js/docx/smoke/api_stroke/to_json.js')
+    json = JSON.parse(docx.elements[1].nonempty_runs.first.text)
+    stroke = json['graphic']['spPr']['ln']['fill']['fill']['color']['color']['rgba']
```

Comment:
Are you sure that checking the color values in the set Fill checks the Stroke value?

Reply:
Yes, I am sure. color data is in this path